rustls/src/msgs/mod.rs

```diff
@@ -12,6 +12,7 @@ pub mod deframer;
 pub mod enums;
 pub mod fragmenter;
 #[allow(non_camel_case_types)]
+#[macro_use]
```

Comment:
I think it's better to specifically import the required macros by name, which I think is the idiomatic style starting with 2018 edition.